Reading out goal frame

DoernerD opened the pull request:
Reading out the frame from the goal and looking up the corresponding TF

working in _gt for the time being

DoernerD committed:
Remove switch in bringup

DoernerD committed:
Merge branch 'humble' of github.com:smarc-project/smarc2 into humble

DoernerD committed:
Reading waypoint frame header from goal message

DoernerD committed:
Integrating mission state

DoernerD committed:
Merge branch 'humble' of github.com:smarc-project/smarc2 into humble

DoernerD committed:
Merge branch 'humble' of github.com:smarc-project/smarc2 into humble

DoernerD committed:
Merge branch 'bt_launch_gt_flag' of github.com:KKalem/smarc2 into humble

DoernerD committed:
Debugging WPF state machine (wip...)

DoernerD committed:
Init basic depth pitch control

DoernerD committed:
Basic depth control with VBS

Controller can use some tuning, but it does the job for now.

DoernerD commented:
Now with VBS depth control

DoernerD committed:
Added basic pitch control

DoernerD committed:
Messages required for the controllers

KKalem commented on messages/control_msgs/msg/Topics.msg:
Arent all these dr/ topics already defined in dr messages already? Having the same topic string in multiple places defeats the purpose of these...

DoernerD committed:
Added Control Convenience Topics

DoernerD committed:
Update ConvenienceView

Integrated the control convenience topics in the ConveienceView Node

DoernerD committed:
README

DoernerD committed:
Active Dive Control

- Diving using either LCG/VBS or thrusters
- Signs need clarification
- Bit of logic of what to use when

- Adjusted bringup script numbering

KKalem committed:
update sim submodules

KKalem committed:
Merge branch 'humble' of github.com:DoernerD/smarc2 into david_server

KKalem committed:
fixed converter service

KKalem committed:
Fixed goto_wp server not realizing it reached its goal

DoernerD committed:
Merge pull request #1 from KKalem/david_server

Fixed the WP follower

DoernerD committed:
Basic active diving

DoernerD committed:
Update bringup for proper tmux numbering

DoernerD commented:
Basic waypoint following works now, static + dynamic diving needs to be integrated into the BT
